app/jenkins-for-jira-ui/src/App.tsx

```diff
@@ -54,7 +54,7 @@ export const environmentSettings = {
 
 const AppContainer = styled.div`
 	color: #172B4D;
-	margin: 24px 36px 24px 0;
+	margin: ${token('space.200')} ${token('space.400')} ${token('space.300')} ${token('space.0')};
```

Comment:
Had to tweak this a little so figured I'd convert to using the Atlassian space styling while I was at it.